File: api/src/main/java/com/cami_rehberi/api/dto/MekanFotoDto.java
package com.cami_rehberi.api.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MekanFotoDto {
    private String url;
    private Boolean isKapakFoto;
    private Integer sira;
}

File: muftuluk/lib/data/models/mekan_foto_dto.dart
class MekanFotoDto {
  final String url;
  final bool isKapakFoto;
  final int sira;

  MekanFotoDto({
    required this.url,
    required this.isKapakFoto,
    required this.sira,
  });

  factory MekanFotoDto.fromJson(Map<String, dynamic> json) {
    return MekanFotoDto(
      url: json['url'],
      isKapakFoto: json['isKapakFoto'],
      sira: json['sira'],
    );
  }
}
